engine/src/main/java/org/hibernate/validator/internal/util/Version.java

```diff
@@ -17,15 +14,9 @@
  * @author Kevin Pollet &lt;kevin.pollet@serli.com&gt; (C) 2012 SERLI
  */
 public final class Version {
-	private static final Pattern JAVA_VERSION_PATTERN = Pattern.compile( "^(?:1\\.)?(\\d+)$" );
```

Comment:
I don't like keeping code which is unused. It's still in the repo should there be the need to resurrect it.